### PR TITLE
Add Help Entry & Infra

### PR DESCRIPTION
**Pull request by kingthorin**
Add the files necessary for a basic help entry, addressing one more task bullet from zaproxy/zaproxy#3742

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: retirejs-zap-plugin/pom.xml [hunk 1]
```diff
@@ -32,6 +32,9 @@
                 <directory>src/main/resources-filtered</directory>
                 <filtering>true</filtering>
             </resource>
+            <resource>
+                <directory>src/main/javahelp</directory>
```
Comment (h3xstream) [inline retirejs-zap-plugin/pom.xml]: I'm not sure we want to support javahelp resource.
Is this something that will be show in ZAP UI?

Comment (h3xstream) [reply retirejs-zap-plugin/pom.xml]: I was not familiar with javahelp. It seems to be a deprecated project : https://github.com/javaee/javahelp

Comment (kingthorin) [reply retirejs-zap-plugin/pom.xml]: It is shown within ZAP. This is the standard setup for all existing addons.

Comment (kingthorin) [reply retirejs-zap-plugin/pom.xml]: It's also used to generate content for zaproxy.org (https://www.zaproxy.org/docs/desktop/addons/)

**Comment by kingthorin** [comment]
@h3xstream Good with this?

File: retirejs-zap-plugin/pom.xml (hunk 1 above)
Comment (psiinon) [reply retirejs-zap-plugin/pom.xml:36]: It is recommended for all ZAP add-ons - its how our users expect to find info about add-ons and we also use it to generate the online content.

Comment (thc202) [reply retirejs-zap-plugin/pom.xml:36]: And it is linked from the marketplace page: https://www.zaproxy.org/addons/

**Pull request closed without merging**